src/WebJobs.Script.Scaling/AzureEnvironments.cs

```diff
@@ -0,0 +1,200 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Win32;
+
+namespace Microsoft.Azure.WebJobs.Script.Scaling
+{
+    public static class AzureEnvironments
```

Comment:
We have similar class in Kudu implementing IEnvironments.   I don't what to call Constants given it is actually env variables.   I will rename it to AzureEnvironmentSettings.